numpy/_pytesttester.py

```diff
@@ -160,6 +160,12 @@ def __call__(self, label='fast', verbose=1, extra_argv=None,
             "-W ignore::UserWarning:cpuinfo",
             ]
 
+        # When testing matrices, ignore their PendingDeprecationWarnings
+        # Note that this does not work for pytest == 3.8.0
```

Comment:
This comment is not quite true, right? I mean, it does work, except if there's a `pytest.ini` file setting warnings filters of their own.

Reply:
Good point, I'll update it.